source/content/drupal-10.md

```diff
@@ -12,15 +12,29 @@ product: [composer]
 integration: [--]
 ---
 
-Drupal 10 is coming in December 2022! We've been busy conducting early testing on the Pantheon platform, and have developed a process for creating a new Drupal 10 site on our platform.
+Drupal 10 is coming in December 2022! This page will be updated frequently with the latest information and instructions for Drupal 10 on Pantheon.,
+
+## Current Status of Drupal 10 at Pantheon
+
+| Scenario | Status | Documentation |
+|---|---|---|
+| Create a Drupal 10 site on Pantheon | Available | [Create a Drupal 10 Site](/drupal-10#create-a-drupal-10-site)|
+| Upgrade a Drupal 9 site to Drupal 10 manually | Available | [Upgrade from Drupal 9 to Drupal 10](/drupal-10#upgrade-a-drupal-9-site-to-drupal-10) |
+| Create Drupal 10 site from Pantheon Dashboard | Planned availability January 2023 | |
+| Upgrade a Drupal 9 site to Drupal 10 using the [Terminus Conversion Tools plugin](https://github.com/pantheon-systems/terminus-conversion-tools-plugin) | Early Access | Use the following command: `terminus conversion:upgrade-d10` |
+
+
+## Create a Drupal 10 Site
+
+We've been busy conducting early testing on the Pantheon platform, and have developed a process for creating a new Drupal 10 site on our platform.
 
 <Alert title="Warning" type="danger" >
 
 These instructions were developed using beta versions of Drupal 10, and are for educational purposes only.  This page will be updated as production versions become available.
```

Comment:
Have we tested with rc2 yet? Seems that we should be able to downgrade this warning / upgrade our confidence at this point.